src/index.js

```diff
@@ -13,6 +13,7 @@ let Monitor = require('./monitor');
  * {
  *   project: '...',
  *   patchGlobal:  true,
+ *   bailOnUnhandledRejection: false,
```

Comment:
That's all well and good for documentation for Jonas, but for the rest of the world, it needs to be documented in the README